src/codeu/chat/server/fileWriter.java

```diff
@@ -0,0 +1,98 @@
+package codeu.chat.server;
+
+import java.util.concurrent.BlockingQueue;
+import java.lang.Runnable;
+
+import codeu.chat.common.Writeable;
+import codeu.chat.util.Serializers;
+
+import java.io.FileOutputStream;
+import java.io.FileInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import codeu.chat.common.User;
+
+import codeu.chat.util.Logger;
+
+public class fileWriter implements Runnable {
+
+    private static BlockingQueue<Writeable> queue;
```

Comment:
Avoid global state like this unless it's absolutely necessary. In the case of the server/controller, you could make this a member variable, and then in server, pass an instance of FileWriter to the controller via its constructor.